server/controllers/event.js

```diff
@@ -391,21 +391,22 @@ function findOneWithSlug(slug, req, res, next) {
     ],
   })
     .then(data => {
-      if (!data.isPublic && req.user.isVisitor) {
-        next(
-          new APIError(
-            'Requested resource is not public',
-            httpStatus.FORBIDDEN
+      // get and pass config vars into function
```

Comment:
This can later be refactored to make use of the new middleware: https://github.com/adzialocha/hoffnung3000/pull/91/files#diff-676f5a85de7da7b2eebe4669b06bfc46